libs/dh/metering-point/feature-search/src/components/dh-create-metering-point.component.ts

```diff
@@ -104,15 +119,21 @@ export class DhCreateMeteringPoint {
       validityDate: dhMakeFormControl<Date | null>(this.today, Validators.required),
       status: dhMakeFormControl({ value: 'NEWLY_CREATED', disabled: true }),
       meteringPointId: dhMakeFormControl('', Validators.required),
```

Comment:
nitpick: Could you set up some basic validation regarding the length and/or type of input?

Reply:
Good idea. I have it locally and didn't want to make the PR bigger. So it come as a follow-up PR.